modules/gdscript/tests/scripts/analyzer/warnings/unused_private_class_variable.gd

```diff
@@ -0,0 +1,10 @@
+# GH-72135
+
+var _a
+@warning_ignore("unused_private_class_variable")
+var _b
+@warning_ignore("unused_private_class_variable") var _c
```

Comment:
Was the lack of a newline here intentional, to make sure the annotation applied to `_c`? :)

Reply:
Yes, I added this case for reinsurance. I can remove it if you think it's not needed.

Reply:
I think it's fine! :)